components/admin_console/jobs/table.test.tsx

```diff
@@ -80,7 +81,7 @@ describe('components/admin_console/jobs/table', () => {
             status: 'warning',
             type: 'data_retention',
         }],
-    };
+    } as unknown as Props;
```

Comment:
Instead of using `as Props`, could you pass mocked objects for the missing fields? If it's because we don't have mocks for `jobs`, you could override the type for that, but we'd rather not use `as Props` in these tests since it'll hide when stuff fails from `props` being incorrect